crates/ty_python_semantic/resources/mdtest/annotations/callable.md

```diff
@@ -381,7 +381,7 @@ def f_okay(c: Callable[[], None]):
     if hasattr(c, "__qualname__"):
         c.__qualname__  # okay
         # `hasattr` only guarantees that an attribute is readable.
-        # error: [invalid-assignment] "Object of type `Literal["my_callable"]` is not assignable to attribute `__qualname__` on type `(() -> None) & <Protocol with members '__qualname__'>`"
+        # error: [invalid-assignment] "Object of type `Literal["my_callable"]` is not assignable to attribute `__qualname__` of type `(() -> None) & <Protocol with members '__qualname__'>`"
```

Comment:
Hmm, I'm not sure whether I prefer "on" or "of" here. But either way -- could we push the decision on whether to make the change to a followup PR, rather than this one? It makes it hard to evaluate the mypy_primer diff, since this changes so many error messages are changing as a result of this change